router/router.go

```diff
@@ -2236,7 +2251,7 @@ func (rt *HandleT) Setup(backendConfig backendconfig.BackendConfig, jobsDB jobsd
 	rt.routerResponseTransformStat = stats.NewTaggedStat("response_transform_latency", stats.TimerType, stats.Tags{"destType": rt.destName})
 
 	rt.transformer = transformer.NewTransformer()
-	rt.transformer.Setup()
+	rt.transformer.Setup(rt.netClientTimeout)
```

Comment:
Please adapt `trackStuckDelivery` method too.

We shouldn't have two different ways of initializing each timeout. Either pass them both of them as arguments or initialize them both during loadConfig. Same applies for their scope (global var vs struct field)

Reply:
Agreed. Done